mmv1/products/datastream/ConnectionProfile.yaml

```diff
@@ -36,12 +36,18 @@ examples:
     vars:
       connection_profile_id: 'my-profile'
   - !ruby/object:Provider::Terraform::Examples
-    name: 'datastream_connection_profile_bigquery_private_connection'
+    name: 'datastream_connection_profile_postgresql_private_connection'
```

Comment:
This will need `pull_external: true` to use the random provider!

Reply:
QQ: When the tests are triggered? I saw it's triggered long time after a new commit is added